lib/helper/messagesHelper.js

```diff
@@ -2,13 +2,16 @@
 
 const giphy = require('./giphy');
 const weather = require('./weather');
+const text = require('./text');
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>'require' is not defined.